lib/events.js

```diff
@@ -394,7 +396,28 @@ EventEmitter.prototype.listeners = function listeners(type) {
   return ret;
 };
 
-EventEmitter.listenerCount = function(emitter, type) {
+EventEmitter.prototype.listenerCount = function listenerCount(type) {
+
+  if (typeof type !== 'string') {
+    throw new TypeError('type must be a valid string');
```

Comment:
The wording here may be a bit misleading since we don't actually validate `type` when it *is* a string. Maybe just say `type must be a string`?